Cargo.toml

```diff
@@ -9,23 +9,29 @@ documentation = "https://docs.rs/fuse-rust"
 repository = "https://github.com/Blakeinstein/fuse-rust"
 homepage = "https://github.com/Blakeinstein/fuse-rust"
 categories = ["algorithms", "text-processing", "gui"]
-keywords = ["weighted-search", "search", "bitap", "fuzzy-search", "fuzzy-matching"]
+keywords = [
+  "weighted-search",
+  "search",
+  "bitap",
+  "fuzzy-search",
+  "fuzzy-matching",
+]
 description = """
 Fuse is a super lightweight library which provides a simple way to do fuzzy searching.
 Fuse-Rust is a port of Fuse-Swift, written purely in rust.
 """
 
 [dependencies]
-crossbeam-utils = { version = "0.8.14", optional = true }
+crossbeam-utils = { version = "0.8", optional = true }
+rayon = { version = "1", optional = true }
```

Comment:
Also why are we using version 1? rayon is at `1.10`

Reply:
It is to integrate it better with other crates. 1 is 1.x according to semver